src/main/graphql/GetClientFileMetadata.graphql

```diff
@@ -0,0 +1,5 @@
+query getClientFileMetadata($fileId: UUID!) {
+    getClientFileMetadata(fileId: $fileId) {
```

Comment:
TDR-874 talks about calling a new endpoint called `file` rather than this query, since it's over-specific. My (still slightly shaky) understanding of GraphQL design is that you have a single endpoint for an entity (in this case `file`) and all of the clients call that endpoint with the subset of the fields they need.

For consignments, we're moving towards using `getConsignment` for everything rather than having specific endpoints. (Though it should be called `consignment` rather than the very Java-y `getConsignment`.) It would be good to do the same thing for files. Let me know if it would be helpful to have a chat about it.

Reply:
Ah, OK. I misunderstood what was wanted in the card.